mundane-bytes/src/main/scala/com/ambiata/mundane/bytes/Buffer.scala

```diff
@@ -65,17 +65,18 @@ object Buffer {
     wrapArray(b.bytes, b.offset + o, b.length - o)
   }
 
-  def reset(b: Buffer): Unit =
+  def reset(b: Buffer): Buffer = {
```

Comment:
I was thinking about this some more and decided it was the wrong behaviour. Happy to drop it if you prefer the old version.